internal/testgen/gen_error_nil.go

```diff
@@ -76,6 +80,10 @@ func (g ErrorNilTestsGenerator) TemplateData() any {
 			{Fn: "Equal", Argsf: "nil, nil"},
 			{Fn: "NotEqual", Argsf: "err, err"},
 			{Fn: "NotEqual", Argsf: "nil, nil"},
+			{Fn: "Empty", Argsf: "err.Error()"},    // Requested by https://github.com/Antonboom/testifylint/issues/138
+			{Fn: "NotEmpty", Argsf: "err.Error()"}, // Requested by https://github.com/Antonboom/testifylint/issues/138
+			{Fn: "Zero", Argsf: "err.Error()"},     // Requested by https://github.com/Antonboom/testifylint/issues/138
+			{Fn: "NotZero", Argsf: "err.Error()"},  // Requested by https://github.com/Antonboom/testifylint/issues/138
```

Comment:
These one are not the one requested by Anton I think

He asked for err.String so the [stringer](https://pkg.go.dev/fmt#Stringer) interface, not the [Error](https://go.dev/ref/spec#Errors) one.

These one are valid Error that the error-nil could/should catch, to be discussed of course.

Reply:
@Antonboom what did you mean with "err.String()" and what should be done with `Empty(err.Error())`

Reply:
The pattern is used https://github.com/search?q=language%3Ago+%22assert.Empty%28t%2C+err.Error%28%29%29%22&type=code

I found nothing that detects it

Reply:
anyway @mmorel-35  , I'm fine if these are not added, first and marked as to-do later, as you said it was a partial support

Reply:
> what did you mean with "err.String()"

It just was a typo, thanks.

> should error-nil catch it ?

I think doesn't, because this is completely difference assertions. Nil error != empty message of not-nil error.